### PR TITLE
Triage

### PR DESCRIPTION
**Pull request by roidelapluie**

### CODE REVIEW

**Comment by vStone** [comment]
A lot of merges in this pull request _wheep_

File: manifests/setup/mod.pp [hunk 1]
```diff
@@ -4,20 +4,25 @@
...
 
-  apache::confd {'mod':
-    confd        => $::apache::setup::mod::confd,
-    order        => $::apache::setup::mod::order,
-    load_content => '',
-    warn_content => '',
-    includes     => $::apache::setup::mod::includes,
-    purge        => $::apache::setup::mod::purge,
+  # skip mod.d inclusion for Centos/RHEL7
+  if $::operatingsystemmajrelease != 7 {
```
Comment (vStone) [inline manifests/setup/mod.pp:16]: You should probabably check for redhat-ish operating systems first.

File: templates/config/os/centos6-ssl.conf [hunk 2]
```diff
@@ -43,10 +43,12 @@ SSLPassPhraseDialog  builtin
 SSLSessionCache         shmcb:/var/cache/mod_ssl/scache(512000)
 SSLSessionCacheTimeout  300
 
+<% if @operatingsystemmajrelease.to_i < 7 %>
```
Comment (vStone) [inline templates/config/os/centos6-ssl.conf:46]: No parameter/facts related logic in the templates please.

**Comment by roidelapluie** [comment]
We will discuss this on Friday.. wheter we spend time on this or on moving to puppetlabs-apache

**Comment by vStone** [comment]
k. nevertheless, this 'everything-in-one' pull request will not get merged like this :)

**Pull request closed without merging**